core/bindings.rs

```diff
@@ -18,16 +18,6 @@ use crate::ops::OpCtx;
 use crate::JsRealm;
 use crate::JsRuntime;
 
-#[derive(Copy, Clone, Eq, PartialEq)]
```

Comment:
Let's keep this one. The bindings code is very complicated right now and I prefer to have an explicit mode until we have a chance to re-visit it.

Reply:
I'm wondering if we can address that complication here. Question: What's the purpose of `&& !will_snapshot` in https://github.com/denoland/deno/blob/7a6753ae77dfdf9745033d620b68f9b314d2631c/core/bindings.rs#L136-L137 and can we remove it? That would be the final separation between 'used snapshot' and 'will snapshot' and op initialisation will no longer depend on the latter. The comment above doesn't mention it. I'll try removing it

Reply:
I've gone with
```rust
enum InitMode {
  New,
  FromSnapshot,
}
```